cmd/kubestatus/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/datawire/teleproxy/pkg/k8s"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	var st = &cobra.Command{
+		Use:           "kubestatus <kind>",
+		Short:         "kubestatus",
+		Long:          "kubestatus - get and set status of kubernetes resources",
```

Comment:
I wrote yesterday:

I think this should be 

```Go
Short: "manage a local Docker registry and Kubernetes cluster",
```

`Short` is for a 1-liner human description of the program, `Long` is for a paragraph description.